playwright-e2e/dsm/component/tabs/onc-history-tab.ts

```diff
@@ -0,0 +1,114 @@
+import {expect, Locator, Page} from '@playwright/test';
+import OncHistoryTable from '../tables/onc-history-table';
+import {waitForResponse} from '../../../utils/test-utils';
+import Select from '../../../dss/component/select';
+import Button from '../../../dss/component/button';
+
+
+export default class OncHistoryTab {
+  private readonly oncHistoryTable = new OncHistoryTable(this.page);
+
+  constructor(private readonly page: Page) {}
+
+  public get table(): OncHistoryTable {
+    return this.oncHistoryTable;
+  }
+
+  public async openFilesOrderModal(): Promise<void> {
+    await this.page.getByText('Change order/files in Bundle').click();
+  }
+
+  public async selectFilesOrderAndDownload(nameOfDocument: string, order: number): Promise<void> {
+    const modalContent = this.filesOrderModalBodyContent(nameOfDocument);
+
+    await expect(modalContent, `Files order modal is not visible`).toBeVisible();
+
+    const orderSelection = new Select(this.page, {root: modalContent});
+    await orderSelection.selectOption(order.toString());
+  }
+
+  public async downloadPDFBundleAfterOrder(): Promise<void> {
+    const downloadPDFBundleBtnLocator = this.filesOrderModalFooter;
+    await expect(downloadPDFBundleBtnLocator, `Download PDF Bundle button is not visible`).toBeVisible();
+
+    const downloadPDFBundleBtn = new Button(this.page, {root: downloadPDFBundleBtnLocator, label: 'Download PDF Bundle'});
+    await downloadPDFBundleBtn.click();
+
+    await waitForResponse(this.page, {uri: 'downloadPDF'});
+    const downloadFinishedText = this.page.getByText('Download finished.');
+    await expect(downloadFinishedText, 'Downloading finished is not visible').toBeVisible();
+  }
+
+  public async downloadPDFBundle(): Promise<void> {
+    const downloadPDFBundleBtn = this.downloadPDFBundleButton;
+    await this.downloadPDFFlow(downloadPDFBundleBtn);
+  }
```

Comment:
This can be reduced to one-line fuction. But a new function with one-line is not needed at all.

Reply:
why not needed?